default-application-catalog/trivy-operator-app.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps.kubermatic.k8c.io/v1
+kind: ApplicationDefinition
+metadata:
+  name: trivy-operator
+spec:
+  description: Trivy-Operator is Kubernetes-native security toolkit.
+  method: helm
+  versions:
+  - template:
+      source:
+        helm:
+          chartName: trivy-operator
+          chartVersion: 0.15.1
+          url: https://aquasecurity.github.io/helm-charts/
+    version: 0.15.1
+  # During the applicationdefinition creation time if the defaultValues is updated, then it is render for override during the application creation process from UI.
+  # By default, it is set empty.
+  defaultValues:
+    trivy:
+      ignoreUnfixed: true
```

Comment:
I think for this case, I would prefer a short explanation as a comment here, why we chose to set `ignoreUnfixed` to true

Reply:
Added comment for `ignoreUnfixed`